suites/manage/ocp/test_ocs_284.py

```diff
@@ -0,0 +1,111 @@
+import os
+import logging
+import ocs.defaults as defaults
+from ocs.ocp import OCP
+from utility import utils, templating
+from ocsci.enums import TestStatus
+
+log = logging.getLogger(__name__)
+
+RBD_SC_YAML = os.path.join("ocs-deployment", "storage-manifest.yaml")
+TEMP_SC_YAML_FILE = '/tmp/tmp-storage-manifest.yaml'
+
+RBD_PVC_YAML = os.path.join("ocs-deployment", "PersistentVolumeClaim.yaml")
+TEMP_PVC_YAML_FILE = '/tmp/tmp-persistentVolumeClaim.yaml'
+
+occli = OCP(kind='service', namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+
+
+def create_rbd_cephpool(poolname, storageclassname):
+    """
+    Creates rbd storage class and ceph pool
+    """
+    data = {}
+    data['metadata_name'] = poolname
+    data['storage_class_name'] = storageclassname
+    data['blockpool_name'] = poolname
+    _templating = templating.Templating()
+    tmp_yaml_file = _templating.render_template(RBD_SC_YAML, data)
+
+    with open(TEMP_SC_YAML_FILE, 'w') as fd:
+        fd.write(tmp_yaml_file)
+        log.info(f"Creating RBD pool and storage class")
+    assert occli.create(TEMP_SC_YAML_FILE)
+    log.info(f"RBD pool: {poolname} storage class: {storageclassname} "
+             "created successfully")
+    log.info(TEMP_SC_YAML_FILE)
+
+
+def create_pvc(pvcname):
+    """
+    Creates a pvc with an user provided name
+    """
+    data = {}
+    data['pvc_name'] = pvcname
+    _templating = templating.Templating()
+    tmp_yaml_file = _templating.render_template(RBD_PVC_YAML, data)
+    with open(TEMP_PVC_YAML_FILE, 'w') as fd:
+        fd.write(tmp_yaml_file)
+        log.info(f"Creating a pvc with name {pvcname}")
+
+    # Code from line:49 to line:76 will be converted to a library so that
+    # it can be consumed whenever required
+    import shlex
+    import subprocess
+
+    def run_ocp_cmd(cmd, **kwargs):
```

Comment:
this function shouldn't be in a test file and it already exists in ocs.ocp

Reply:
I created this function to get the error and do the validation using the collected error. I see Raz has submitted a PR with these changes in the existing function. Will use it once it is merged.